app/src/main/java/com/arctouch/movies/splash/view/SplashActivity.java

```diff
@@ -0,0 +1,37 @@
+package com.arctouch.movies.splash.view;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+
+import com.arctouch.movies.R;
+import com.arctouch.movies.base.view.BaseActivity;
+import com.arctouch.movies.home.view.HomeActivity;
+import com.arctouch.movies.splash.presenter.SplashActivityPresenter;
+import com.arctouch.movies.splash.presenter.SplashPresenter;
+
+/**
+ * Created by marcelo.alves on 06/09/17.
+ */
+
+public class SplashActivity extends BaseActivity<SplashPresenter> implements SplashView {
```

Comment:
If I open the app by accident and close it with the back button while it's still in the Splash screen, it still loads the app after 3 seconds, even though I've already closed it... What could be done to avoid it?

Reply:
Create a method into `Presenter` that removes the current `Runnable` into `Handler` . This method need to be called into `onPause` from `Activity`.